tests/test_primod/test_preprocessing_ribamod.py

```diff
@@ -141,7 +143,7 @@ def test_ribamod_write_toml(
     mf6_modelname, mf6_model = get_mf6_gwf_modelnames(mf6_bucket_model)[0]
     mf6_river_packages = get_mf6_river_packagenames(mf6_model)
 
-    driver_coupling = RibaModPassiveDriverCoupling(
```

Comment:
Maybe it’s better to change the package, now you’re changing the complete test? Is there still passive coupling being tested here?

Reply:
I am not sure I understand this comment. 
The bucket model does not seem to have drainage, only a river package, which raises an exception now.

Reply:
Changed the river package in the model to a drain and changed the active drivercoupling back to a passive drivercoupling as it was. Also removed the corresponding api-package associated with the river.
Explicit assert on the exception and the message introduced in this issue.